src/utils/VideoRelevanceManager.ts

```diff
@@ -5,11 +5,35 @@ import { globalEm } from './globalEm'
 // constant used to parse seconds from creation
 export const NEWNESS_SECONDS_DIVIDER = 60 * 60 * 24
 
+type VideoRelevanceManagerLoops = {
+  fullUpdateLoopTime: number
+  scheduledUpdateLookTime: number
```

Comment:
I think you meant `Loop` instead of `Look`

```suggestion
  scheduledUpdateLoopTime: number
```